src/middleware.ts

```diff
@@ -22,7 +22,13 @@ async function loadShipsCookie(
       const ships = await fetchShips(slackId, 3)
       response.cookies.set({
         name: 'ships',
-        value: JSON.stringify(ships),
+        value: JSON.stringify(
+          ships.map((s) => {
+            if (s.screenshotUrl.startsWith('data:'))
+              s.screenshotUrl = `replace_me_with_a_non_dataencoded_url_plz`
+            return s
+          }),
+        ),
```

Comment:
Is this to fix data that is currently incorrect in the wild?

Reply:
yes! currently it soft locks users (aka vercel 500) which means they cant ship projects. ofc i can remove this and you guys will have to unlock them manually ;-;

Reply:
Ok amazing - tysm for this!